package.json

```
@@ -170,7 +170,7 @@
     "packageManager": [
       {
         "name": "npm",
-        "version": ">=11.3.0 <=11.6.2",
+        "version": "^11.3.0",
```

Should it not exclude 11.6.3?

```suggestion
        "version": "^11.6.4",
```

---

Why break is a developer has 11.6.2 locally?